pkg/metrics/metrics.go

```diff
@@ -213,6 +233,8 @@ func init() {
 	prometheus.MustRegister(NodeGroupTaintEvent)
 	prometheus.MustRegister(NodeGroupUntaintEvent)
 	prometheus.MustRegister(NodeGroupScaleLock)
+	prometheus.MustRegister(NodeGroupScaleLockDuration)
+	prometheus.MustRegister(NodeGroupScaleLockCheckWasLocked)
```

Comment:
Are you able to add these new metrics to https://github.com/atlassian/escalator/blob/master/docs/metrics.md

Reply:
👍 done